schema/snapcraft.yaml

```diff
@@ -583,29 +488,13 @@ properties:
             items:
               type: string
             default: ['*']
-          snap: # FIXME: This is deprecated, replaced by `prime`, below.
```

Comment:
This should probably be done in its own PR, no?

Reply:
it even has its own bug, let me fix that slip